src/main/java/no/nav/familie/inntektsmelding/integrasjoner/person/PersonInfo.java

```diff
@@ -3,6 +3,14 @@
 import java.time.LocalDate;
 
 import no.nav.familie.inntektsmelding.typer.entitet.AktørIdEntitet;
+import no.nav.pdl.Navn;
 
-public record PersonInfo(String navn, PersonIdent fødselsnummer, AktørIdEntitet aktørId, LocalDate fødselsdato) {
+public record PersonInfo(Navn navn, PersonIdent fødselsnummer, AktørIdEntitet aktørId, LocalDate fødselsdato) {
```

Comment:
Har ikke helt kål på når man lager egne records eller bruker eksisterende. Greit å ta inn navn fra pdl her, eller gjøre om til 3 felt for for/mellom/etter-navn?

Reply:
Jeg ville gjort om til 3 egne felt her, og ikke bruke DTO som tilhører PDL. Også for å unngå så nøstede objekter.